<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finance</class>
 <widget class="QDialog" name="finance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="finance_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>711</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>691</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Dépenses</string>
     </property>
     <widget class="QTabWidget" name="tabWidget_depense">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>651</width>
        <height>261</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>611</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout dépense</string>
        </property>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>date</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>libelle</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>70</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>id employe</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>30</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>montant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_id_r_ajout_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_libelle_r_ajout_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>140</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_montant_r_ajout_3">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="de_date_r_ajout_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>90</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="cb_r_emp_ajout_3">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>70</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_ajoutDepense">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>140</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_depense">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>481</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier par</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_tri_dp">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_36">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recherche_depense">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="excel">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>export excel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="export_2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>140</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>exporter pdf</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>165</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>libelle</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>montant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>id employe</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_r_id_modif_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="de_date_r_ajout_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_r_emp_ajout_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>100</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_libelle_r_ajout_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_montant_r_ajout_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_editDepense">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>170</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;MV Boli&quot;;
color: rgb(185, 142, 42);</string>
        </property>
        <property name="text">
         <string>Identifiant Dépense</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_d_supp">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>70</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_depense">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>140</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>681</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Revenues</string>
     </property>
     <widget class="QTabWidget" name="revenues_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>661</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout revenues</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>135</y>
           <width>55</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>libelle</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>55</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>75</y>
           <width>55</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>date</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>100</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>id employe</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>35</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>montant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_revenue_aj">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="date_revenue_aj">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>80</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="libelle_revenue_aj">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="montant_revenue_aj">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>30</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="id_empl_rev_aj">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>100</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_ajoutRevenue">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>70</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_revenues">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>441</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_38">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier par</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_sort_r">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>20</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_37">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>76</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_r_search">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf_revenues">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>162</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer pdf</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>35</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>175</y>
          <width>55</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>libelle</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>105</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>montant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>90</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>id employe</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_r_id_modif">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="de_date_r_ajout_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_r_emp_ajout_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>100</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="de_date_r_ajout_7">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_montant_r_ajout_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="update_revenue">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>162</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;MV Boli&quot;;
color: rgb(185, 142, 42);</string>
        </property>
        <property name="text">
         <string>Identifiant Revenue</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_supp_r">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>80</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimerRevenue">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>172</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="depenses">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>352</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dépenses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revenues">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>352</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Revenues</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
